indico/modules/events/contributions/forms.py

```diff
@@ -32,3 +32,4 @@ class ContributionForm(IndicoForm):
     duration = TimeDeltaField(_("Duration"), [DataRequired()], default=timedelta(minutes=20),
                               units=('minutes', 'hours'),
                               description=_("The duration of the contribution"))
+    location_data = IndicoLocationField(_("Location"), description=_("Location"))
```

Comment:
Another thought. This would be the first field we have that a description can't be specified. Is that what we really want? I still think it makes sense to have the description to describe what the location is going to be used/mean/whatever the dev want to specify. Shouldn't we use a slightly different style to display the current location and still allow description to be used?